### PR TITLE
add kicking stats to ncaaf players

### LINKED ISSUE #373: Add Kicking table stats for NCAAF players
Currently the `"kicking"` table is not being read for (slightly) more advanced kicking stats for NCAAF players. I have PR coming to fix this so no need to start working.

### PR DESCRIPTION
**Pull request by AidanGlickman**
Fixes #373

Adds kicking stats (attempts and percentage) to NCAAF players.

Not sure how the previous PR #374  broke but here it is again?

### CODE REVIEW

File: sportsreference/ncaaf/player.py [hunk 1]
```diff
@@ -501,10 +507,41 @@ def extra_points_made(self):
...
+        """
+        Returns an ``int`` of the total number of extra points the player
+        attempted.
+        """
+        return self._extra_points_attempted
+
+    @_float_property_decorator
+    def extra_point_percentage(self):
+        """
+        Returns a ``float`` of the percentage of extra points the player made.
```
Comment (roclark) [inline sportsreference/ncaaf/player.py:521]: Looks like this still needs a note about the percentage range (0.0 - 100.0) as with the other attribute below.

File: sportsreference/ncaaf/player.py [hunk 2]
```diff
@@ -501,10 +507,41 @@ def extra_points_made(self):
...
+        """
+        Returns an ``int`` of the total number of field goals the player
+        attempted from any distance.
+        """
+        return self._field_goals_attempted
+
+    @_float_property_decorator
+    def field_goal_percentage(self):
+        """
+        Returns a ``float`` of the percentage (0.0 to 100.0) of field goals
```
Comment (roclark) [inline sportsreference/ncaaf/player.py:544]: Also, looks like there's an extra space at the end after "goals". Sorry for not catching this earlier!

File: sportsreference/ncaaf/player.py (hunk 1 above)
Comment (AidanGlickman) [reply sportsreference/ncaaf/player.py]: done!

File: sportsreference/ncaaf/player.py (hunk 2 above)
Comment (AidanGlickman) [reply sportsreference/ncaaf/player.py]: done!

**Comment by roclark** [comment]
Hey @AidanGlickman, I added a [branch](https://github.com/roclark/sportsreference/compare/patch/add-ncaaf-kicking) which includes updates to the test files to properly include this with the testing suite. If you wouldn't mind, it would be great if you could copy the test file and updates I created and include them with this commit. It looks like everything passes the tests on my end, but I want to make sure you have the proper credit for this, so feel free to copy everything as-is.

Let me know if you have any questions or concerns. Otherwise, once the tests are included, I will merge this and include it with my upcoming release!

**Comment by AidanGlickman** [comment]
Sounds good! I went ahead and added them to the commit. I look forward to contributing more where I can!

**Comment by roclark** [comment]
Thanks again @AidanGlickman! This looks great and I will merge it! Thank you very much for the addition and for the changes! I will work on cutting a new release now so this can be included with all new binary pulls.

**Pull request merged**